app-text/pdfminer/pdfminer-20191020.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7} pypy )
+
+inherit distutils-r1
+
+DESCRIPTION="Python tool for extracting information from PDF documents"
+HOMEPAGE="https://euske.github.io/pdfminer/ https://pypi.org/project/pdfminer/"
+SRC_URI="mirror://pypi/p/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc examples test"
+
+BDEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
```

Comment:
Distutils-r1 already puts this in BDEPEND, so you dont need to...

Reply:
I can't find that in the eclass?

There seem a lot of ebuilds that do add it:
```
$ qgrep -v setuptools | grep BDEPEND | wc -l
64
```

Reply:
Hmm, neither can I. No idea where I've gotten the impression it does (probably some custom eclass or a proposal), but Ive been wrong for a long time! Thanks for checking it out and teaching me here :)